include/SFML/Audio/SoundStream.hpp

```diff
@@ -194,10 +194,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
     bool getLoop() const;
 
 protected:
-    enum
-    {
-        NoLoop = -1 //!< "Invalid" endSeeks value, telling us to continue uninterrupted
-    };
+    // NOLINTNEXTLINE(readability-identifier-naming)
+    static constexpr std::int64_t NoLoop = -1; //!< "Invalid" onLoop return value, telling us to continue uninterrupted
```

Comment:
I changed the documentation to what seems more relevant. No idea what `endSeeks` was referring to.